course/staff_views.py

```diff
@@ -8,7 +8,6 @@
 from django.utils.text import format_lazy
 from django.utils.translation import gettext_lazy as _
 from django.utils.translation import ngettext_lazy as ngettext
-
 from authorization.permissions import ACCESS
```

Comment:
This empty line should stay, it separates library imports from the project imports.